gravityforms-securesubmit.php

```diff
@@ -3,12 +3,12 @@
  * Plugin Name: Heartland Secure Submit Addon for Gravity Forms
  * Plugin URI: https://developer.heartlandpaymentsystems.com/securesubmit
  * Description: Integrates Gravity Forms with SecureSubmit, enabling end users to purchase goods and services through Gravity Forms.
- * Version: 1.4.8
```

Comment:
I think that removing support for non-iframes qualifies as a major change. The version # should reflect this. I recommend using 2.0.0